src/current/v23.2/physical-cluster-replication-monitoring.md

```diff
@@ -101,6 +102,72 @@ We recommend tracking the following metrics:
 
+## Data verification
+
+{{site.data.alerts.callout_info}}
+**This feature is in [preview]({% link {{ page.version.version }}/cockroachdb-feature-availability.md %}).** It is in active development and subject to change.
+
+`SHOW EXPERIMENTAL_FINGERPRINT` allows you to verify that the data transmission and ingestion is working as expected while a replication stream is running. Any checksum mismatch likely represents corruption or a bug in CockroachDB. Should you encounter such a mismatch, contact [Support](https://support.cockroachlabs.com/hc/en-us).
+{{site.data.alerts.end}}
+
+To verify that the data up to a certain point in time is correct on the standby cluster, you can use the current replicated time from the replication job information to run a point-in-time fingerprint on both the primary and standby clusters. This will verify that the transmission and ingestion of the data on the standby cluster, up to that point in time, is correct.
+
+1. Retrieve the current replicated time of the replication job on the standby cluster with [`SHOW VIRTUAL CLUSTER`]({% link {{ page.version.version }}/show-virtual-cluster.md %}):
+
+    {% include_cached copy-clipboard.html %}
+    ~~~ sql
+    SHOW VIRTUAL CLUSTER standbyapplication WITH REPLICATION STATUS;
+    ~~~
+    ~~~
+    id |        name        | data_state  | service_mode | source_tenant_name |                                                source_cluster_uri                                                 | replication_job_id |        replicated_time        |         retained_time         | cutover_time
+    ---+--------------------+-------------+--------------+--------------------+-------------------------------------------------------------------------------------------------------------------+--------------------+-------------------------------+-------------------------------+---------------
+    3  | standbyapplication | replicating | none         | application        | postgresql://{user}:redacted@{node IP}:26257/?options=-ccluster%3Dsystem&sslmode=verify-full&sslrootcert=redacted | 925475002117849089 | 2023-12-15 19:39:18.321455+00 | 2023-12-14 19:16:07.321456+00 |         NULL
+    (1 row)
+    ~~~
+
+    For detail on connecting to the standby cluster, refer to [Set Up Physical Cluster Replication]({% link {{ page.version.version }}/set-up-physical-cluster-replication.md %}#connect-to-the-standby-cluster-system-interface).
+
+1. From the **primary cluster's system interface**, specify a timestamp at or lower than the current `replicated_time` to verify the data:
+
+    {% include_cached copy-clipboard.html %}
+    ~~~ sql
+    SELECT * FROM [SHOW EXPERIMENTAL_FINGERPRINTS FROM VIRTUAL CLUSTER application] AS OF SYSTEM TIME '1702669158089558000.0000000000';
```

Comment:
Perhaps these examples should used the replicated time from the output above?

Reply:
Ah, I swapped in the timestamp as per the output. Sorry — I was messing about with this for a while and forgot to make this all consistent. Thanks!